tools/travis/integration.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -e
+
+# Build script for Travis-CI.
+
+SCRIPTDIR=$(cd $(dirname "$0") && pwd)
+ROOTDIR="$SCRIPTDIR/../.."
+WHISKDIR="$ROOTDIR/../openwhisk"
+UTILDIR="$ROOTDIR/../incubator-openwhisk-utilities"
+
+# run scancode
+cd $UTILDIR
+scancode/scanCode.py $ROOTDIR
+
+# TODO run jshint
+
+# Install OpenWhisk
+cd $WHISKDIR/ansible
+
+ANSIBLE_CMD="ansible-playbook -i environments/local"
+
+$ANSIBLE_CMD setup.yml
+$ANSIBLE_CMD prereq.yml
+$ANSIBLE_CMD couchdb.yml
+$ANSIBLE_CMD initdb.yml
+
+cd $WHISKDIR
+
+./gradlew distDocker
+
+cd $WHISKDIR/ansible
+
+$ANSIBLE_CMD wipe.yml
+$ANSIBLE_CMD openwhisk.yml
+$ANSIBLE_CMD apigateway.yml
+
+# Set Environment
+export OPENWHISK_HOME=$WHISKDIR
+
+# Set up CLI tool used by prereq script
+sudo cp $WHISKDIR/bin/wsk /usr/bin/wsk
```

Comment:
Hum there is no need for the CLI, the integration tests should use env variables, in case you want to use `$HOME/.wskprops` I would say to put the values directly and avoid building the CLI.